roles/tftp-server/files/ec/early_command.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -eufx
+
+# hook - if ec.sh exists on the server, get it and run it.
```

Comment:
I've still never needed this.

Reply:
this -eufx or the early_command?

I still use it - mostly to copy syslog to a box with more tools so I figure out why ansible errored. 
also for providing details for installer bug reports, like: in-target breaks DNS 
 
https://bugs.launchpad.net/ubuntu/+source/debian-installer/+bug/1717038

Hey look, Steve of DC Portland commentated - hi Steve!

Reply:
:)

Reply:
You never need it because I find all the bugs.  You're welcome.  :p 

There might be a simpler way to hook this, like "(wget foo && ./foo) | True" so a failed wget doesn't halt the installer.    Not sure if that is better.